packages/plugin-mongo/src/database.ts

```diff
@@ -22,6 +22,11 @@ export interface Config {
   uri?: string
 }
 
+interface TableConfig<O> {
```

Comment:
Its called `Collection` in mongo, not `Table`.

Reply:
Just for unification.